src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -1443,6 +1449,13 @@ static void perform_stream_op_locked(void* stream_op,
       s->stream_compression_method = GRPC_STREAM_COMPRESSION_IDENTITY_COMPRESS;
     }
 
+    if (s->stream_compression_method !=
+        GRPC_STREAM_COMPRESSION_IDENTITY_COMPRESS) {
+      s->uncompressed_data_size = 0;
+      s->stream_compression_ctx = nullptr;
+      grpc_slice_buffer_init(&s->compressed_data_buffer);
+    }
+
```

Comment:
we've been following the no unnecessary blank line rule

Reply:
Sure removed.